config/webpack.common.js

```diff
@@ -26,19 +24,19 @@ const buildUtils = require('./build-utils');
  *
  * See: https://webpack.js.org/configuration/
  */
-module.exports = function (options) {
+module.exports = function(options) {
   const isProd = options.env === 'production';
   const APP_CONFIG = require(process.env.ANGULAR_CONF_FILE || (isProd ? './config.prod.json' : './config.dev.json'));
 
-  const METADATA = Object.assign({}, buildUtils.DEFAULT_METADATA,options.metadata || {});
+  const METADATA = Object.assign({}, buildUtils.DEFAULT_METADATA, options.metadata || {});
   const GTM_API_KEY = process.env.GTM_API_KEY || APP_CONFIG.gtmKey;
 
   const ngcWebpackConfig = buildUtils.ngcWebpackSetup(isProd, METADATA);
```

Comment:
why don't you delete this line? I thought you replaced ngcwebpack with ngtoolswebpack

Reply:
The config is still used, though. ngcWebpack is just a wrapper around webpack/ngtools and almost all config values are the same.

Reply:
We might want to do some renaming here for clarity, though...

Reply:
thanks!